packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/ClientProvider.cs

```diff
@@ -113,6 +131,12 @@ internal IReadOnlyList<ParameterProvider> GetUriParameters()
             if (_uriParameters is null)
             {
                 _ = Constructors;
+                if (_parent.Value is not null)
```

Comment:
Just curious: Is it still required to have to lookup the parent within a sub-client? Is this gathering the required uri parameters from the parent ?

Reply:
Yes - this is required to populate the uri parameters, without it the code doesn't generate and 'client' is missing from the uriParameters list.